plugins/test/meson.build

```diff
@@ -46,3 +52,26 @@ shared_module('fu_plugin_invalid',
     plugin_deps,
   ],
 )
+
+shared_module('fu_plugin_test_ble',
```

Comment:
Should this plugin rename to `test_bluetooth` or `test_bluez`?

Reply:
Given other comment, this name selection seems accurate.  However I want to point out it's still pointless to compile this plugin when you're compiling without `-Dbluez=true`.  I think this plugin shared module declaration should have a guard around that still.